html/captive-portal/templates/mfa/TOTP.html

```diff
@@ -0,0 +1,24 @@
+<!-- START TOTP.html -->
+
+<div class="c-card o-layout o-layout--center u-padding u-padding@tablet">
+    <div class="o-layout__item u-1/1 u-2/3@tablet u-3/5@desktop">
+[% IF exist %]
+        <form action="/mfa" method="post">
+            <div class="input-container">
+                <label for="otp">[% i18n("OTP") %]</label>
+                <input id="otp" name="otp" title="OTP" type="tel" inputmode="numeric" pattern="[0-9]*" minlength="4" maxlength="6" required autocorrect="off" autocapitalize="off" class="text-center letter-expand" />
+            </div>
+            <button type="submit" name="submit" class="c-btn c-btn--primary u-1/1 u-margin-top" disabled>
+                [% i18n("Continue") %]
+            </button>
+        </form>
```

Comment:
We should add a footnote that says what is expected from the user, example:
![image](https://user-images.githubusercontent.com/3857942/134208893-195f976f-292a-47cc-81d3-304656400179.png)